samples/exemplar/src/main/kotlin/com/squareup/exemplar/dashboard/frontend/FrontendIndexAction.kt

```diff
@@ -0,0 +1,627 @@
+package com.squareup.exemplar.dashboard.frontend
+
+import kotlinx.html.ButtonType
+import kotlinx.html.InputType
+import kotlinx.html.a
+import kotlinx.html.button
+import kotlinx.html.div
+import kotlinx.html.footer
+import kotlinx.html.form
+import kotlinx.html.h1
+import kotlinx.html.h2
+import kotlinx.html.h3
+import kotlinx.html.header
+import kotlinx.html.id
+import kotlinx.html.img
+import kotlinx.html.input
+import kotlinx.html.label
+import kotlinx.html.li
+import kotlinx.html.nav
+import kotlinx.html.option
+import kotlinx.html.p
+import kotlinx.html.role
+import kotlinx.html.select
+import kotlinx.html.span
+import kotlinx.html.ul
+import kotlinx.html.unsafe
+import misk.config.AppName
+import misk.turbo.buildHtml
+import misk.turbo.turbo_frame
+import misk.web.Get
+import misk.web.PathParam
+import misk.web.ResponseContentType
+import misk.web.actions.WebAction
+import misk.web.dashboard.AdminDashboardAccess
+import misk.web.dashboard.HtmlLayout
+import misk.web.mediatype.MediaTypes
+import wisp.deployment.Deployment
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/**
+ * Example page from Tailwind UI
```

Comment:
This example is a _lot_ to consume for anyone unfamiliar with this tech (e.g. me). We should consider also offering a simpler example that shows how to do something a little more basic.